src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -136,14 +157,23 @@ internal LibraryView AddLibraryView()
             var view = new LibraryView(model);
 
             var browser = view.Browser;
+            this.browser = browser;
             sidebarGrid.Children.Add(view);
             browser.RegisterJsObject("controller", this);
             RegisterResources(browser);
 
             view.Loaded += OnLibraryViewLoaded;
+            browser.SizeChanged += Browser_SizeChanged;
```

Comment:
browser.LoadError => will this be helpful?

Reply:
cool, I'll check this out, we might at the minimum be able to log a better error to the dynamo console.